src/main/java/uk/gov/hmcts/sscs/domain/notify/EventType.java

```diff
@@ -8,6 +8,7 @@
     APPEAL_LAPSED("appealLapsed"),
     APPEAL_RECEIVED("appealReceived"),
     APPEAL_WITHDRAWN("appealWithdrawn"),
+    APPEAL_DORMANT("appealDormant"),
```

Comment:
Does this need a new case definition? The appeal dormant event does not have a callback, as no notification was needed but think we need to add that in to handle this new scenario